playground/lib/home/pages/sheet_page_with_text_field.dart

```diff
@@ -17,6 +17,7 @@ class SheetPageWithTextField {
       isButtonEnabledNotifier.value = textEditingController.text.isNotEmpty;
     });
     return WoltModalSheetPage(
+      resizeToAvoidBottomInset: true,
```

Comment:
`resizeToAvoidBottomInset` in the  `wolt_modal_sheet_default_theme_data.dart` defaults to `true` in this PR.

So do we need to specify this `true` value?

Reply:
It is a left over since I was trying to test the `ƒalse`. Good catch! Thanks <3 Fixed here: 20e03259844c0fadea3e3f9690add0246025aeb8